Quartz version 2.5.1 error

### Bug description 🐞

After upgrading to quartz 2.5.1 the logs are showing the following:

```
2025-12-07T22:27:45.839Z ERROR 1 --- [_MisfireHandler] o.s.s.quartz.LocalDataSourceJobStore     : Error retrieving the misfired trigger: DEFAULT.TerrakubeV2_ModuleRefresh_3c8d06a9-5077-4328-b02c-2adf1bca2c43

org.quartz.JobPersistenceException: Couldn't retrieve trigger: No record found for selection of Trigger with key: 'DEFAULT.TerrakubeV2_ModuleRefresh_3c8d06a9-5077-4328-b02c-2adf1bca2c43' and Scheduler: 'schedulerFactoryBean' and Delegate: class org.quartz.impl.jdbcjobstore.SimpleTriggerPersistenceDelegate
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1546) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.recoverMisfiredJobs(JobStoreSupport.java:975) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.doRecoverMisfires(JobStoreSupport.java:3191) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.manage(JobStoreSupport.java:3941) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.run(JobStoreSupport.java:3960) ~[quartz-2.5.1.jar:2.5.1]
Caused by: org.quartz.impl.jdbcjobstore.NoRecordFoundException: No record found for selection of Trigger with key: 'DEFAULT.TerrakubeV2_ModuleRefresh_3c8d06a9-5077-4328-b02c-2adf1bca2c43' and Scheduler: 'schedulerFactoryBean' and Delegate: class org.quartz.impl.jdbcjobstore.SimpleTriggerPersistenceDelegate
        at org.quartz.impl.jdbcjobstore.SimpleTriggerPersistenceDelegate.loadExtendedTriggerPropertiesFromResultSet(SimpleTriggerPersistenceDelegate.java:108) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.SimpleTriggerPersistenceDelegate.loadExtendedTriggerProperties(SimpleTriggerPersistenceDelegate.java:97) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.handleTrigger(StdJDBCDelegate.java:1945) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectTrigger(StdJDBCDelegate.java:2083) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1543) ~[quartz-2.5.1.jar:2.5.1]
        ... 4 common frames omitted

2025-12-07T22:27:45.840Z ERROR 1 --- [_MisfireHandler] o.s.s.quartz.LocalDataSourceJobStore     : Error retrieving the misfired trigger: DEFAULT.TerrakubeV2_InactiveJobs

org.quartz.JobPersistenceException: Couldn't retrieve trigger: No record found for selection of Trigger with key: 'DEFAULT.TerrakubeV2_InactiveJobs' and Scheduler: 'schedulerFactoryBean' and Delegate: class org.quartz.impl.jdbcjobstore.CronTriggerPersistenceDelegate
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1546) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.recoverMisfiredJobs(JobStoreSupport.java:975) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.doRecoverMisfires(JobStoreSupport.java:3191) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.manage(JobStoreSupport.java:3941) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.run(JobStoreSupport.java:3960) ~[quartz-2.5.1.jar:2.5.1]
Caused by: org.quartz.impl.jdbcjobstore.NoRecordFoundException: No record found for selection of Trigger with key: 'DEFAULT.TerrakubeV2_InactiveJobs' and Scheduler: 'schedulerFactoryBean' and Delegate: class org.quartz.impl.jdbcjobstore.CronTriggerPersistenceDelegate
        at org.quartz.impl.jdbcjobstore.CronTriggerPersistenceDelegate.loadExtendedTriggerPropertiesFromResultSet(CronTriggerPersistenceDelegate.java:110) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.CronTriggerPersistenceDelegate.loadExtendedTriggerProperties(CronTriggerPersistenceDelegate.java:96) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.handleTrigger(StdJDBCDelegate.java:1945) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectTrigger(StdJDBCDelegate.java:2083) ~[quartz-2.5.1.jar:2.5.1]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1543) ~[quartz-2.5.1.jar:2.5.1]
        ... 4 common frames omitted
```

### Steps to reproduce

Use version 2.29.0-beta.3

### Expected behavior

_No response_

### Example repository

_No response_

### Anything else?

_No response_

--- a/api/pom.xml
+++ b/api/pom.xml
@@ -27,7 +27,7 @@
         <rest-assured.version>5.5.6</rest-assured.version>
         <postgresql.version>42.7.8</postgresql.version>
         <snakeyaml.version>2.5</snakeyaml.version>
-        <quartz.version>2.5.1</quartz.version>
+        <quartz.version>2.5.2</quartz.version>
         <aws-sdk.version>2.39.6</aws-sdk.version>
         <gcp-libraries-bom.version>26.72.0</gcp-libraries-bom.version>
         <jjwt.version>0.13.0</jjwt.version>

